tests/components/composer.test.ts

```diff
@@ -429,7 +429,15 @@ describe("ranges and highlights", () => {
 });
 
 describe("composer", () => {
-  test("starting the edition with enter, the composer should have the focus", async () => {
+  test("grid composer is not visible when not editing", async () => {
+    expect(model.getters.getEditionMode()).toBe("inactive");
+    const gridComposerEl = fixture.querySelector(".o-grid-composer") as HTMLDivElement;
+    expect(gridComposerEl.style.zIndex).toBe("-1000");
```

Comment:
Can't we use a less barbaric way to hide the input btw ? Or does display `hidden` and things like that don't work with IME ?

Reply:
I need to check this properly tomorrow, I'm fairly sure it was the issue, yes ^^

Reply:
I confirm that using `display: none` is a no go ^^